### ISSUE TITLE
Training the custome dataset but get poor performance?

### ISSUE THREAD

**Issue by KRYSTALLJY** [JDAI-CV/fast-reid#308]
I combine CUHK03, DukeMTMC and Market as a new dataset, and train the network. the performance is bad. Do you know how to improve it? Loss converges to about 8 (clas_loss:8, and triplet loss 0.1). The rank-1 is about 73%. And the rank-1 is 73% at 120epochs, and degrades to 54% at epoch 200. but the loss is decreasing.

**Comment by L1aoXingyu** [comment]
You can try to tune the hyperparameter, such as batch size and learning rate, by yourself.
